task_manager.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+import sys
+import json
+import rc_util
+from rc_rmq import RCRMQ
+from datetime import datetime
+
+task = 'task_manager'
+
+args = rc_util.get_args()
+logger = rc_util.get_logger(args)
+
+record = {
+    'uid': -1,
+    'gid': -1,
+    'email': '',
+    'last_update': datetime.now(),
+    'request': {
+        'uid_resolve': None
```

Comment:
can you change the task name to `get_next_uid_gid` instead of `uid_resolve ` here.

Reply:
fixes in 39f3f08